src/App.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { solid } from '@fortawesome/fontawesome-svg-core/import.macro';
+
```

Comment:
- [ ]  *Your Project is getting Compiled with Errors*
![meow](https://user-images.githubusercontent.com/104627775/190595801-0d88a30c-5451-4eff-a483-bf1694a25687.PNG)

Reply:
@shuja-shah 
This error is because you haven't set up the linters correctly, please make sure you do that and check it again